modules/app/src/main/resources/admin/tools/main/main.js

```diff
@@ -39,7 +41,13 @@ exports.getParams = function () {
             locale: admin.getLocales()
         }),
         launcherPath: admin.getLauncherPath(),
-        configServiceUrl: portal.serviceUrl({service: 'config'})
+        configServiceUrl: portal.url({
+            path: `/admin/${app.name}/main/_/${app.name}/config`,
+        }),
+        toolBaseUrl: portal.url({
+            path: `/admin/${app.name}/main`,
+        }),
```

Comment:
Can we avoid repeating `/admin/${app.name}/main/`? Make a `const appUrl = '/admin/${app.name}/main/';`